### ISSUE TITLE
When {pkg} and {pkg}-xx come from the same FreeBSD port's different flavors, they should be in one metapackage

### ISSUE THREAD

**Issue by yurivict** [repology/repology-rules#128]
Example: ```erkale``` and ```erkale-serial```.

**Comment by AMDmi3** [comment]
This case is merged, each other case should be reported individually.
